changelog/markdown-default.dd

```diff
@@ -0,0 +1,3 @@
+DDoc Markdown support has been enabled by default
+
+If you encounter issues with this transition, use `-revert=markdown` and report an issue.
```

Comment:
Actually, I don't think this description is enough, because:
- Let's not kid ourselves, most users don't use `-preview`. So re-stating some informations on it (where the documentation can be found, and an overview of the features / changes) is IMO important.
- It's also important to have more than a one-liner for such an important feature. I know many users have asked for it over the year, and we should be a bit more proactive in marketing the good changes we're doing.
- It's not a usual practice, so this is only a suggestion, but I would add a thank you note to @dgileadi for championing this.

Reply:
Fair enough. I'm really lazy with changelog entries. I updated this one.

> It's not a usual practice, so this is only a suggestion, but I would add a thank you note to @dgileadi for championing this.

Great idea! We should make this the usual practice ;-)